### PR TITLE
Uncomment log trace

### PR DESCRIPTION
**Pull request by alexlucaci**
### Description

add a description of your changes here...

### Rationale

tell us why we need these changes...

### Example

add an example CLI or API response...

### Changes

Notable changes: 
* add each change in a bullet point here
* ...

### CODE REVIEW

File: miner/worker.go [hunk 1]
```diff
@@ -852,28 +852,30 @@ LOOP:
...
 		case errors.Is(err, core.ErrNonceTooLow):
 			// New head notification data race between the transaction pool and miner, shift
-			//log.Trace("Skipping transaction with low nonce", "sender", from, "nonce", tx.Nonce())
+			log.Trace("Skipping transaction with low nonce", "sender", from, "nonce", tx.Nonce())
 			txs.Shift()
 
 		case errors.Is(err, core.ErrNonceTooHigh):
 			// Reorg notification data race between the transaction pool and miner, skip account =
-			//log.Trace("Skipping account with hight nonce", "sender", from, "nonce", tx.Nonce())
+			log.Trace("Skipping account with hight nonce", "sender", from, "nonce", tx.Nonce())
```
Comment (Mister-EA) [inline miner/worker.go:878]: There is a typo here.

File: miner/worker.go [hunk 2]
```diff
@@ -852,28 +852,30 @@ LOOP:
...
 		if tx.Protected() && !w.chainConfig.IsEIP155(env.header.Number) {
-			//log.Trace("Ignoring reply protected transaction", "hash", tx.Hash(), "eip155", w.chainConfig.EIP155Block)
+			log.Trace("Ignoring reply protected transaction", "hash", tx.Hash(), "eip155", w.chainConfig.EIP155Block)
 			txs.Pop()
 			continue
 		}
 		// Start executing the transaction
 		env.state.Prepare(tx.Hash(), env.tcount)
 
+		from, _ := types.Sender(env.signer, tx)
```
Comment (emailtovamos) [inline miner/worker.go:862]: Why no error check here?

**Review by emailtovamos — CHANGES_REQUESTED** [review]

File: miner/worker.go (hunk 2 above)
Comment (alexlucaci) [reply miner/worker.go:862]: I don't know if we should throw an error from here. I copy/paste what was written on line 851

**Review by Mister-EA — APPROVED** [review]

File: miner/worker.go (hunk 2 above)
Comment (brilliant-lx) [reply miner/worker.go:862]: It was commented in line:847, the tx should have been check in the txpool, no they skip the error check here.
```
// Error may be ignored here. The error has already been checked
// during transaction acceptance is the transaction pool.
```

**Review by brilliant-lx — APPROVED** [review]

**Review by emailtovamos — APPROVED** [review]

**Pull request merged**